.evergreen/aws

```diff
@@ -45,6 +45,16 @@ when 'assume-role'
   puts "AWS_SESSION_TOKEN=#{credentials.session_token}"
   #puts "EXPIRATION=#{credentials.expiration.utc.strftime('%Y%m%dT%H%M%SZ')}"
   puts
+when 'assume-web-role'
+  arn = ARGV.shift
+  token_file = ARGV.shift
+
+  orchestrator = AwsUtils::Orchestrator.new(**options)
+  credentials = orchestrator.assume_role_with_web_identity(arn, token_file)
+  puts "AWS_ACCESS_KEY_ID=#{credentials.access_key_id}"
+  puts "AWS_SECRET_ACCESS_KEY=#{credentials.secret_access_key}"
+  puts "AWS_SESSION_TOKEN=#{credentials.session_token}"
+  puts
```

Comment:
I know it implies a larger refactoring here, but what if the bodies of each of these `when` clauses were refactored to separate methods? It would make the enclosing `case` statement more concise, which (for me, selfishly, as a newbie 😄) would be a win, since it would be easier to scan and see what the various intentions are.

Reply:
Done